### PR TITLE
DEV: Bump selenium-webdriver version to fix system spec running

### PR DESCRIPTION
**Pull request by markvanlan**
We can no long user Webdriver - https://github.com/SeleniumHQ/selenium/issues/11066. Bumping selenium-webdriver did the trick, as well as manually setting the user_agent for mobile system specs. Unsure what changed to make this necessary, but it is necessary to get the app to boot in mobile view.

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -255,7 +252,7 @@ def fab!(name, &blk)
 
     SiteSetting.provider = TestLocalProcessProvider.new
 
-    WebMock.disable_net_connect!(allow_localhost: true, allow: [Webdrivers::Chromedriver.base_url])
+    WebMock.disable_net_connect!(allow_localhost: true)
```
Comment (pmusaraj) [inline spec/rails_helper.rb:255]: I wonder whether this is part of the issue with the failing specs? Maybe we need to allow some kind of extra base_url?

Comment (pmusaraj) [reply spec/rails_helper.rb:255]: Hmm, no, looks like that's only used to update Webdrivers, `Webdrivers::Chromedriver.base_url` points to `https://chromedriver.storage.googleapis.com`.

Comment (markvanlan) [reply spec/rails_helper.rb:255]: Yeah not sure about that yet, but mobile specs aren't getting the correct user_agent so trying to specify that first. Not seeing anything online regarding emulation for chromedriver changing however.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -356,6 +353,9 @@ def synchronize(seconds = nil, errors: nil)
         .new(logging_prefs: { "browser" => browser_log_level, "driver" => "ALL" })
         .tap do |options|
           options.add_emulation(device_name: "iPhone 12 Pro")
+          options.add_argument(
+            '--user-agent="--user-agent="Mozilla/5.0 (Windows Phone 10.0; Android 4.2.1; Microsoft; Lumia 640 XL LTE) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Mobile Safari/537.36 Edge/12.10166"',
+          )
```
Comment (pmusaraj) [inline spec/rails_helper.rb:358]: Do we need both an Android and an iOS agent here? Since device is iPhone, maybe we stick with just that user agent?

**Review by pmusaraj — APPROVED** [review]
Approving this pre-emptively, seems like the right approach, feel free to merge if you're confident it fixes CI @markvanlan

File: spec/rails_helper.rb (hunk 2 above)
Comment (markvanlan) [reply spec/rails_helper.rb:358]: Pushed another commit -  I copy/pasted the wrong user agent. I've changed it to the iOS agent, which is what we are emulating.

**Pull request merged**